skbio/io/registry.py

```diff
@@ -244,6 +245,31 @@ def add_format(self, format_object):
         else:
             self._text_formats[name] = format_object
 
+    def remove_format(self, format):
+        """Remove a format from the IORegistry.
+
+        Parameters
+        ----------
+        format : str or Format object
+            The format to be removed from the registry. It can be a name or the Format
+            object itself.
+
+        """
+        if isinstance(format, skbio.io.registry.Format):
+            # print(format)
+            format = format.name
+        success = False
+        for lookup in self._lookups:
+            try:
+                if format in lookup:
+                    del lookup[format]
+                    success = True
+                    break
+            except KeyError():
```

Comment:
Can a test of membership be explicit?

Reply:
Actually, I don't think this would arise because the membership test is done?

Reply:
Good point! The try-except block I don't think was necessary so I removed it.